src/parser/jobs/smn/modules/GeneralCDDowntime.js

```diff
@@ -1,9 +1,11 @@
 import ACTIONS from 'data/ACTIONS'
 import CooldownDowntime from 'parser/core/modules/CooldownDowntime'
 
-//Although the trance cooldown is 55s, it is better in many cases
-//to hold for a 60s alignment, so some downtime per use must be allowed.
-const TRANCE_ALLOWED_DOWNTIME = 5000
+//The trance cooldowns are both 55s.  However, in order to maintain overall
+//alignment with the 1 minute buff cycle, FBT should be held 10s per use
+//(for a 55+65 = 120s cycle).  Due to the way cooldown tracking works in
+//XIVA, both must have the same allowed downtime since they are in a cooldown group.
```

Comment:
sounds like an issue that needs to be resolved.

this is fine for now, but it'd be good to get the underlying issue resolved such that it's not required

Reply:
Other options I considered around this:

- Under the proposed change to core cooldowns, I'd probably change to a 5s offset.  This would calculate as if they were used every 1 minute (which is the recommended timing for buff alignments).
- Not mark them as a group and give them both a "2 minute cooldown".  They aren't really a group in the typical sense (with normal groups, like RDM displacement/engagement, you can use only one of them all the time.), but they do trigger each other's cooldowns.  You would also need to give a firstUseOffset to Firebird Trance to account for needing to use Dreadwyrm Trance first.
- Don't use core cooldown logic, and make the custom usage calculation logic specifically for the trance to allow the 55/65 split.

Reply:
All sound like sane solutions to me, whichever is a go-ahead for you works.